src/platform/ESP32/PlatformManagerImpl.cpp

```diff
@@ -57,7 +57,11 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
 {
     CHIP_ERROR err;
     wifi_init_config_t cfg;
+    uint8_t ap_mac[6];
 
+    esp_fill_random(ap_mac, sizeof(ap_mac));
+    /* Bit 0 of the first octet of MAC Address should always be 0 */
+    ap_mac[0] = ap_mac[0] & 0xFE;
```

Comment:
nit:
```suggestion
    ap_mac[0] &= ~0x01;
```

Reply:
Done!

Reply:
With the change, the restyler is now stuck at `restyled Expected — Waiting for status to be reported`, which passed previously :sob: Same is the case with the my other PRs